assets/app/view/game/route_selector.rb

```diff
@@ -56,6 +55,7 @@ def render
           store(:selected_route, route, skip: true)
         end
 
+        render_halts = false
```

Comment:
render_halts = route.num_halts

Reply:
Ok

Reply:
Ok

Reply:
Wait, render_halts is calculated across *all* routes. If any of the routes has halts, then render_halts is set true.